### PR TITLE
qpdfview: update to 0.5.0

### PR DESCRIPTION
**Pull request by zen0bit**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x64 glibc)

### CODE REVIEW

File: srcpkgs/qpdfview/template [hunk 1]
```diff
@@ -12,8 +12,8 @@ short_desc="Tabbed document viewer"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://launchpad.net/qpdfview"
-distfiles="https://launchpad.net/qpdfview/trunk/${version}/+download/qpdfview-${version}.tar.gz"
-checksum=cc642e7fa74029373ca9b9fbc29adc4883f8b455130a78ad54746d6844a0396c
+distfiles="https://launchpad.net/qpdfview/trunk/${version}.0/+download/qpdfview-${version}.tar.gz"
```
Comment (abenson) [inline srcpkgs/qpdfview/template:15]: Why not set version as 0.5.0?

Comment (zen0bit) [reply srcpkgs/qpdfview/template:15]: Because full link is...

https://launchpad.net/qpdfview/trunk/0.5.0/+download/qpdfview-0.5.tar.gz

Comment (classabbyamp) [reply srcpkgs/qpdfview/template:15]: upstream calls the release 0.5.0: https://launchpad.net/qpdfview/+announcement/30778

Comment (zen0bit) [reply srcpkgs/qpdfview/template]: As you wish...

File: srcpkgs/qpdfview/template [hunk 2]
```diff
@@ -12,8 +12,8 @@ short_desc="Tabbed document viewer"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://launchpad.net/qpdfview"
-distfiles="https://launchpad.net/qpdfview/trunk/${version}/+download/qpdfview-${version}.tar.gz"
-checksum=cc642e7fa74029373ca9b9fbc29adc4883f8b455130a78ad54746d6844a0396c
+distfiles="https://launchpad.net/qpdfview/trunk/${version}/+download/qpdfview-0.5.tar.gz"
```
Comment (leahneukirchen) [inline srcpkgs/qpdfview/template:15]: use ${version} here

Comment (zen0bit) [reply srcpkgs/qpdfview/template:15]: it wouldnt match then

version=**0.5.0**
file name qpdfview-**0.5**.tar.gz no **.0** here

I think better leave as is for now
then
Could be changed with new release with proper naming convention

or maybe patch for now?

Comment (zen0bit) [reply srcpkgs/qpdfview/template:15]: `version=0.5.0`
`distfiles="https://launchpad.net/qpdfview/trunk/${version}/+download/qpdfview-0.5.tar.gz"`

or

`version=0.5`
`distfiles="https://launchpad.net/qpdfview/trunk/${version}.0/+download/qpdfview-${version}.tar.gz"`

File name version number differ from upstream version number 
How to handle that?

**Pull request merged**